src/Components/Components/src/Rendering/ComponentState.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
```

Comment:
Is this still needed? Maybe it is, but I can't see anything obvious here in the diff that appears to need it.

Reply:
No, it's VS being VS